File: 2609/2609.java
// Backjoon Online Judge No.2609
// Java

package problem;

import java.util.Scanner;

public class No2609{
    
    // 재귀gcd함수를 이용하였다.
    static int gcd(int x, int y){
        return (y > 0) ? gcd(y, x % y) : x;
    }

    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        int A = 0, B = 0;
        A = sc.nextInt();
        B = sc.nextInt();

        System.out.println(gcd(A, B));
        System.out.print(A * B / gcd(A, B));
        
        sc.close();
    }
}

File: 2609/2609.c
// Backjoon Online Judge
// C
#include<stdio.h>

// gcd함수를 재귀로 구현해보았다.
int gcd(int x, int y){
    return (y > 0) ? gcd(y, x % y) : x;
}

int main(void){
    int A = 0, B = 0;
    scanf("%d %d", &A, &B);

    printf("%d\n%d", gcd(A, B), (A * B / gcd(A, B)));
    return 0;
}